code/controllers/subsystem.dm

```diff
@@ -35,6 +35,12 @@
 
 	var/static/list/failure_strikes //How many times we suspect a subsystem type has crashed the MC, 3 strikes and you're out!
 
+	//yogs start -- loading time stuff
+	var/static/total_loading_points_progress = 0 // How much progress we've made in loading all the subsystems so far.
```

Comment:
make them autodocs since you went through the trouble of commenting your code

Reply:
It'd be the first autodoc committed in ages but uhh... sure?

Reply:
gotta start somewhere

thankfully we have good habits of doi... wait